### DISCUSSION TITLE
v5 版本如何自动重载配置？

### DISCUSSION THREAD

**Discussion by smallst** [IrineSistiana/mosdns#787] [Q&A]
v5 版本删除了auto reload。但是我希望通过shell 来定时重载（更新后的）配置文件，该怎么做呢？是否有相应的api？ 是否只能通过 `/etc/init.d/mosdns restart` ?

**Comment by eskimokk** [comment]
同问

**Reply by smallst** [reply]
目前我选择了 restart 😂

**Reply by eskimokk** [reply]
我定时会更新分流数据，目前看起来只能restart

**Reply by luzhaohui** [reply]
v5 是退步，我还继续 v4.5.3

**Comment by luweijun1992** [comment]
写个shell、更新完域名列表后 restart compose.yaml
